app/main.py

```diff
@@ -1,62 +1,132 @@
-from typing import List
+#!/usr/bin/env python
+# coding: utf-8
 
-from fastapi import FastAPI
-from pydantic import BaseModel
-
-app = FastAPI()
-
-
-class Todo(BaseModel):
-    id: int
-    name: str
-
-
-sample_todo = Todo(id=1, name="Sample todo")
-global_todos: List[Todo] = [sample_todo]
-
-# This is a change
-# Get Todos
-@app.get("/todos")
-async def get_todos():
-    return {"todos": global_todos}
+# In[ ]:
 
+from contextlib import asynccontextmanager
 
-# Get Todo by id
-@app.get("/todos/{todo_id}")
-async def get_todo(todo_id: int):
-    for todo in global_todos:
-        if todo.id == todo_id:
-            return {"todo": todo}
+from typing import Union, Annotated
 
-    return {f"'message': 'No  todo of id: {todo_id}'"}
-
-
-# Create Todo
-@app.post("/todos")
-async def create_todo(new_todo: Todo):
-    global_todos.append(new_todo)
-
-    return {f"'message': 'Todo {new_todo.name} successfully created!'"}
-
-
-# Update Todo
-@app.put("/todos")
-async def update_todo(updated_todo: Todo):
-    for todo in global_todos:
-        if todo.id == updated_todo.id:
-            todo.id = updated_todo.id
-            todo.name = updated_todo.name
-
-            return {f"'message': Updated todo {updated_todo.name}'"}
+from fastapi import FastAPI, Query, Path, BackgroundTasks
+# from fastapi.testclient import TestClient
+from pydantic import BaseModel
+import numpy as np
+import requests
+import appliance
+import InstallAPIcalls
+import time
+from fastapi.middleware.wsgi import WSGIMiddleware
+from InstallGraphing import app as ElectricityGraph
 
-    return {f"'message': 'Todo not found'"}
+app = FastAPI()
 
 
-# Delete Todo by id
-@app.delete("/todos/{todo_id}")
-async def delete_todo(todo_id: int):
-    global global_todos
-    updated_todos = filter(lambda todo: todo.id != todo_id, global_todos)
-    global_todos[:] = list(updated_todos)
 
-    return {"message": "Todo successfully deleted"}
+users = []
+customers = []
+elec_dicts = []
+
+
+# @app.get("/")
+# def read_root():
+#     return {"Hello": "World"}
+
+# client = TestClient(app)
+
+@app.post("/users/")
+async def read_user_bayou(
+        utility: str = "speculoos_power",
+        email: str = "QuickStart@bayou.energy"
+):
+    bayou_user = {
+        "utility": utility,
+        "email": email
+    }
+
+    users.append(bayou_user)
+    return users
+
+
+@app.get("/users/{user_id}")
+async def read_user(user_id: int):
+    return users[user_id]
+
+
+# This is to create a new customer in Bayou
+@app.post("/new_customers/")
```

Comment:
We could probably change this route name to just be `/customers` like the other routes since it's a post route. Then we can change line 78 to a get method from the same `/customers` endpoint.

Reply:
Good call, I'll fix this in the next push (in the new branch). Is just changing "post" to "get" sufficient in the new line?

Reply:
Should do the trick!